### PR TITLE
Check dotnet format in travis

### PR DESCRIPTION
**Pull request by shargon**
This will prevent to push any format error

Related to https://github.com/neo-project/neo/issues/982

### CODE REVIEW

**Comment by shargon** [comment]
@igormcoelho Works as expected, this project require a format

File: tests/neo-vm.Tests/UtDebugger.cs [hunk 1]
```diff
@@ -158,11 +168,16 @@ public void TestBreakPointStepOver()
...
-                /* │  ┌> */ script.Emit(OpCode.NOT);
-                /* │  │  */ script.Emit(OpCode.RET);
-                /* └>X│  */ script.Emit(OpCode.PUSH0);
-                /*   └┘  */ script.Emit(OpCode.RET);
+                /* ┌     CALL 
+                   │  ┌> NOT
+                   │  │  RET
+                   └>X│  PUSH0
+                     └┘  RET */
+                script.EmitJump(OpCode.CALL, 5);
```
Comment (shargon) [inline tests/neo-vm.Tests/UtDebugger.cs]: Thanks for preserve the comments 😬👍

**Comment by shargon** [comment]
why pass with wrong lf?

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+root = true
+
+# Every file
+
+[*]
+insert_final_newline = true
+trim_trailing_whitespace = true
+charset = utf-8
+end_of_line = lf
+#indent_style = tab  # TODO
```
Comment (erikzhang) [inline .editorconfig]: Must remove `TODO` before merging.

**Comment by shargon** [comment]
Anyone could review this pull request?

**Pull request merged**